pkg/blob/controllerserver.go

```diff
@@ -667,6 +683,103 @@ func (d *Driver) DeleteBlobContainer(ctx context.Context, subsID, resourceGroupN
 	})
 }
 
+// CopyBlobContainer copies a blob container in the same storage account
+func (d *Driver) CopyBlobContainer(ctx context.Context, subsID, resourceGroupName, accountName, accountKey, srcContainerName, dstContainerName string, secrets map[string]string) error {
+	if srcContainerName == "" {
+		return fmt.Errorf("srcContainerName is empty")
+	}
+	if dstContainerName == "" {
+		return fmt.Errorf("dstContainerName is empty")
+	}
+	if acquired := d.volumeLocks.TryAcquire("copy_" + dstContainerName); !acquired {
+		return status.Errorf(codes.Aborted, volumeOperationAlreadyExistsFmt, dstContainerName)
+	}
+	defer d.volumeLocks.Release("copy_" + dstContainerName)
+	return wait.ExponentialBackoff(d.cloud.RequestBackoff(), func() (bool, error) {
```

Comment:
it's not necessary to do `ExponentialBackoff` here since csi driver would do the backoff

Reply:
ok, I have delete to return error directly.